docker/daemon.go

```diff
@@ -26,17 +29,26 @@ import (
 	"github.com/docker/docker/utils"
 )
 
-var (
-	daemonCfg   = &daemon.Config{}
-	registryCfg = &registry.Options{}
-)
+func presentInHelp(usage string) string { return usage }
+func absentFromHelp(string) string      { return "" }
+
+// NewDaemonCli returns a pre-configured daemon CLI
+func NewDaemonCli() *DaemonCli {
+	daemonFlags = cli.Subcmd("daemon", nil, "Enable daemon mode", true)
```

Comment:
I know this is from the old code but `Enable daemon mode` sounds odd. Almost like its turning on daemon mode in addition to its real/main task.  What about `Start the Docker daemon`  ?  This is repeated down below too.